framework/source/class/qx/ui/basic/Image.js

```diff
@@ -514,7 +514,7 @@ qx.Class.define("qx.ui.basic.Image",
         }
       }
       else if (ResourceManager.has(source)) {
-        var highResolutionSource = this._findHighResolutionSource(source);
+        var highResolutionSource = qx.util.ResourceManager.getInstance().findHighResolutionSource(source);
```

Comment:
`qx.util.ResourceManager.getInstance()` has already been saved as `ResourceManager`, maybe shorten it here